app/src/main/java/org/mozilla/focus/ui/theme/FocusTypography.kt

```diff
@@ -21,6 +21,11 @@ val metropolis = FontFamily(
     Font(R.font.metropolis_bold, FontWeight.Bold),
     Font(R.font.metropolis_semibold, FontWeight.SemiBold),
 )
+val roboto = FontFamily(
```

Comment:
I see the question about roboto still unanswered - https://github.com/mozilla-mobile/focus-android/issues/7500#issuecomment-1232807537
Making a mark of that here to check later on if to update this or not.
Maybe @mcarare knows more from Fenix about metropolis / roboto / device default fonts.

Reply:
On Fenix, there was a UX/ Product decision to get rid of the Metropolis font and use the default device font instead.
There is no need to add the Roboto font, just let Android to use it's default font (it is also expected that some device manufacturers have their own fonts).

! If we also want to remove it from Focus it would involve no new designs using it.

Reply:
Done I deleted roboto font